website/docs/docs/cloud/dbt-cloud-ide/git-commit-signing.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Git commit signing"
+description: "Learn how to sign your Git commits when using the IDE for development."
+sidebar_label: Git commit signing
+---
+
+# Git commit signing <Lifecycle status="Enterprise" />
+
+To improve identity protection/impersonation and enhance security, you can sign your Git commits when pushing them to the repository. Using your signature, a Git provider can cryptographically verify it and mark the commit as "verified", giving people increased confidence about the origin of the commit. 
+
+You can configure dbt Cloud to sign your Git commits when using the IDE for development. To set up, enable the feature in dbt Cloud, generate a GPG keypair, and upload the public key to your Git provider to use for signature verification.  
```

Comment:
we should say "follow the flow to generate a keypair" bc they don't need to generate their own keypair - we manage that for them/

Reply:
updated in this commit: https://github.com/dbt-labs/docs.getdbt.com/pull/6324/commits/73d097b0d7784ef3645b0e715ee364184aaf2471